plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSystemTables.java

```diff
@@ -103,4 +112,344 @@ public void testPropertiesTable()
             assertUpdate("DROP TABLE IF EXISTS " + tableName);
         }
     }
+
+    @Test
+    public void testPartitionsTable()
+    {
+        String tableName = "test_simple_partitions_table";
```

Comment:
Please add `randomNameSuffix` to avoid flaky tests due to name conflicts. Same for others.